android/src/main/AndroidManifestNew.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+</manifest>
```

Comment:
Is this file needed? I can't find it in the upgrade helper.

Reply:
bootstraping a new library using the cli actually creates this additional manifest, that's why I added